mne_bids/read.py

```diff
@@ -76,6 +76,42 @@ def _read_raw(raw_fpath, electrode=None, hsp=None, hpi=None, config=None,
     return raw
 
 
+def _handle_participants_reading(participants_fname, raw,
+                                 subject, verbose=None):
+    participants_tsv = _from_tsv(participants_fname)
+    subjects = participants_tsv['participant_id']
+    row_ind = subjects.index(subject)
+
+    # mapping subject information back to mne-python
+    sex_options = {'n/a': 0, 'M': 1, 'F': 2}
+    hand_options = {'n/a': 0, 'R': 1, 'L': 2, 'A': 3}
+
+    # map age to a random date of that year
+    age = participants_tsv['age'][row_ind]
+    if age != 'n/a':
+        year = datetime.now().year - int(age)
+        birthday = (year, 1, 1)
```

Comment:
I don't get this part. Say I recorded data from subj X on 2015-01-01 ... and at that time of recording, the age of X was 50. Today it's 2020-04-17, and I want to read that data of X again ... so I read the age (50) and subtract that from current year (2020), which means the birthday is 1970-01-01, when actually it's sometime in 1965?